hicommonwealth/commonwealth
# Updated admin/user onboarding sliders position on discussion pages

## Link to Issue
Closes: https://github.com/hicommonwealth/commonwealth/issues/7888

## Description of Changes
Added logic to make admin/user onboarding cards/slider, to scroll the whole page, when mouse is hovered and scrolled on the cards/slider.

## "How We Fixed It"
N/A

## Test Plan
1. Enable `FLAG_USER_ONBOARDING_ENABLED`
2. Create a new account and visit `/discussions` page of any community
3. Verify that you can scroll the full page (even when hovering over the User onboarding cards/slider)
4. Now create a new community and visit the `/discussions` page of that community
5. Verify you see the admin onboarding slider/cards and that you can scroll the full page (even when hovering over the Admin onboarding cards/slider) -- If you still see the User onboarding cards/slider, then dismiss them and reload the page to view admin onboarding slider.
6. Verify the discussion page looks good with/without any sliders (admin/user-onboarding) on small screens.

## Deployment Plan
N/A

## Other Considerations
N/A
